source/update.rst

```diff
@@ -56,7 +56,7 @@ Semi-automatic upgrade
          "spec": {
             "crVersion":"{{{release}}}",
             "image": "percona/percona-server-mongodb:{{{mongodb44recommended}}}",
-            "backup": { "image": "percona/percona-server-mongodb-operator:{{{release}}}-backup" },
+            "backup": { "image": "percona/percona-server-mongodb-operator:{{{pbmrecommended}}}-backup" },
             "pmm": { "image": "percona/pmm-client:{{{pmm2recommended}}}" }
```

Comment:
we need to use backup image 
percona/percona-backup-mongodb:1.7.0
not our custom build